### PR TITLE
Add gamingboots.json

### PR DESCRIPTION
**Pull request by gamingboots**

### CODE REVIEW

File: domains/gamingboots.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A description",
+    "repo": "https://github.com/gamingboots/gamingboots.github.io",
+    "owner": {
+      "username": "gamingboots",
+      "email": "gamngboots19@gmail.com",
+      "twitter": "your-twitter-username"
```
Comment (MaskDuck) [inline domains/gamingboots.json:7]: this is **NOT** a valid twittter username.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/gamingboots.json (hunk 1 above)
Comment (SX-9) [reply domains/gamingboots.json:7]: lol epic username if you don't have a twitter acc then just delete line 7

Comment (MaskDuck) [reply domains/gamingboots.json:7]: :mmlol: big brain

**Review by MaskDuck — APPROVED** [review]
btw rickrolling in ~~pycord~~ github cool =)

**Pull request merged**